### DISCUSSION TITLE
Not Found for url ...  when try bundle with github auth_token

### DISCUSSION THREAD

**Discussion by linfanluntan** [Project-MONAI/MONAILabel#1297] [General]
As below:

File "C:\Users\FullerLabAI\anaconda3\lib\site-packages\monai\bundle\scripts.py", line 434, in _get_all_bundles_info
    resp.raise_for_status()
  File "C:\Users\FullerLabAI\anaconda3\lib\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://api.github.com/repos/Project-MONAI%5Cmodel-zoo/releases

when try: export MONAI_ZOO_AUTH_TOKEN=<Your github auth token>, got no export command

**Comment by SachidanandAlle** [comment]
On windows.. you can use environment way to add new variables defined.. in this case `MONAI_ZOO_AUTH_TOKEN`

Or another way is set.. you can find couple of choices when you search online..
example: https://phoenixnap.com/kb/windows-set-environment-variable

**Comment by daniel-a-alexander** [comment]
you can use the set command on windows instead of export.
 I am having the same issue. I set the token, verifies in the command line read out, but I get the same error.

**Comment by SachidanandAlle** [comment]
I see you have hit the same issue on windows.. that's a bug from monai side which is fixed only in dev branch.
There is a url encoding issue due to hardcoded path character..

`https://api.github.com/repos/Project-MONAI%5Cmodel-zoo/releases`

The solution is, uninstall monai and use github way.. set PYTHONPATH variable to github monai checkout project directory.
Or you can install latest monai-weekly if you don't want github way..

**Comment by SachidanandAlle** [comment]
https://github.com/Project-MONAI/MONAI/pull/5787
